.travis.yml

```diff
@@ -101,7 +101,7 @@ before_install:
   ## Avoid bug on OSX (https://github.com/travis-ci/travis-ci/issues/6307)
   # This prevents the build from failing with:
   # `/Users/travis/build.sh: line 109: shell_session_update: command not found`
-  - if [ "$TRAVIS_OS_NAME" = "osx" ]; then rvm get head; fi
+  # - if [ "$TRAVIS_OS_NAME" = "osx" ]; then rvm get head; fi
```

Comment:
Travis failure seems unrelated since the linux build fails to find Doxygen, but can you confirm?

Reply:
Yes the failure is unrelated; see #2039. The test doesn't get far along enough to actually test the code, though.